core/controllers/base.py

```diff
@@ -275,6 +275,18 @@ def render_json(self, values):
         json_output = json.dumps(values, cls=utils.JSONEncoderForHTML)
         self.response.write('%s%s' % (feconf.XSSI_PREFIX, json_output))
 
+    def _get_logout_url(self, redirect_url_on_logout):
+        """Prepares and returns logout url which will be handled
+        by LogoutPage handler.
+
+        Args:
+            redirect_url_on_logout: str. URL to redirect to on logout.
+
+        Returns:
+            str. logout url.
```

Comment:
str. The logout URL.

(Start all descriptions with capital letters and have them form full sentences. Be consistent in terms of using URL (as you've done above) vs url.)

Reply:
done.